app/client/views/application/application.html

```diff
@@ -76,15 +76,26 @@
                  ng-model="user.profile.description">
         </div>
 
-        <div class="field">
+        <div class="field" ng-class="{error: user.profile.essay.length>1500}">
           <label>
               What would you like to learn or get out of HackUCI?
           </label>
           <textarea ng-model="user.profile.essay"
             name="essay"
-            placeholder="Please enter your response here (100 chars min, 1500 max)">
+            placeholder="Please enter your response here (100 chars min, 1500 max)" 
+            ng-class="{'error':user.profile.essay.length>1500}">
           </textarea>
-          <span>{{user.profile.essay.length}}/1500</span>
+          <div class="essay-error-flexcontainer">
+            <div class="error-message" >
+              <span ng-show="user.profile.essay.length>1500" class="error-message">The essay length must be between 100 and 1500 characters.</span>
```

Comment:
leave a space between the operator, also is there a reason you need to include the `error-message` class in here and the previous line?